File: Prak603/PRAK603-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
int main(void)
{
    int x, isi, baris1, baris2, hasil;
    scanf ("%i %i", &baris1, &baris2);
    int bil1[baris1], bil2[baris2];
    if (baris1 == baris2){
        for (x = 0; x < baris1; x++){
            scanf ("%i", &isi);
            bil1[x] = isi;     
        }
        for (x = 0; x < baris2; x++){
            scanf ("%i", &isi);
            bil2[x] = isi;     
        }
        for (x = 0; x < baris1; x++){
            hasil = bil1[x] * bil2[x];
            printf ("%i ", hasil);     
        }
    }
    else {
        printf ("jumlah tidak sama");
    }
}

File: Prak603/PRAK603-2210817210021-TRISNACAHYAPERMADI.py
b1, b2 = map(int, input().split())
if b1 != b2 :
    print ("jumlah tidak sama")
else :
    bil1 = list(map (int, input().split()))
    bil2 = list(map (int, input().split()))
    m1 = []
    m2 = []
    x = 0
    m1.append(bil1 [x : x + b1])
    m2.append(bil2 [x : x + b2])
    for i in range (0, b1):
        print(m1[0][i] * m2[0][i], end=" ")
